types/okr/check-in-rule.ts

```
@@ -0,0 +1,35 @@
+  appliesTo: string;
+  planningPeriodId: string;
+  timeBased: boolean;
+  achievementBased: boolean;
+  frequency: number;
+  operation: string;
+  action: 'Appreciation' | 'Reprimand';
+  categoryId: string;
+  feedbackId?: string;
+  target?: number;
+  targetDate?: Array<{
+    date: string;          // "monday" - the day this rule applies to
```

ESLint

[error] Delete `·········`

(prettier/prettier)